.github/workflows/build-test.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         git clone https://github.com/Chia-Network/clvm.git --branch=develop --single-branch
         python -m pip install ./clvm
 
-        git clone https://github.com/Chia-Network/clvm_tools.git --branch=develop --single-branch
+        git clone https://github.com/Chia-Network/clvm_tools.git --branch=deserialize_api --single-branch
```

Comment:
This is just for testing the new api, once we're convinced it works with this branch, we can remove it and merge this (and then merge clvm_tools).